ecf/defs/prod00.def

```diff
@@ -3,30 +3,38 @@ extern /prod18/gdas/atmos/post
 extern /prod18/gdas/enkf/post
 #### ecen
 extern /prod18/gdas/atmos/post_processing/jgdas_atmos_chgres_forenkf
+extern /prod06/gdas/atmos/analysis/jgdas_atmos_analysis
+extern /prod06/gfs/atmos/analysis/jgfs_atmos_analysis
+extern /prod06/gdas/enkf/analysis/recenter/ecen
 
 suite prod00
   repeat day 1
   edit ECF_TRIES '1'
   edit CYC '00'
   edit ENVIR 'prod'
-  edit PROJ 'OPS'
-  edit E 'j'
-  edit QUEUE 'prod'
+  edit PROJ 'GFS'
```

Comment:
All of these settings have been changed from ops settings to dev, which is required to run, but not correct for the hand-off.